components/playlist/browser/resources/components/app.tsx

```diff
@@ -7,7 +7,7 @@ import * as React from 'react'
 import { bindActionCreators, Dispatch } from 'redux'
```

Comment:
After changing the state location, I ran formatter so there are a lot of changes but practically this doesn't affect the behavior of the existing web ui.